src/modules/extra/m_ssl_openssl.cpp

```diff
@@ -409,6 +434,9 @@ static BIO_METHOD biomethods =
 	OpenSSL::BIOMethod::destroy, // destroy, does nothing, see function body for more info
 	NULL // callback_ctrl
 };
+#else
+static BIO_METHOD* biomethods = OpenSSL::BIOMethod::alloc();
```

Comment:
I'm not sure if this is correct, it gets executed before the ModuleOpenSSL's constructor which is where openssl is initialized. Move it to the constructor after library initialization, unless it's documented to work like this.